packages/mui-base/src/utils/useAnchorPositioning.ts

```diff
@@ -275,3 +275,19 @@ export function useAnchorPositioning(
     ],
   );
 }
+
+function isRef(param: unknown): param is React.RefObject<any> {
+  return {}.hasOwnProperty.call(param, 'current');
```

Comment:
Why not? It takes less space:
```suggestion
  return param.hasOwnProperty('current');
```

Reply:
It's not a part of this PR anymore